delfin/drivers/dell_emc/unity/rest_handler.py

```diff
@@ -81,44 +83,47 @@ def login(self):
             LOG.error("Login error: %s", six.text_type(e))
             raise e
 
-    def call_with_token(self, url, data, method):
+    def call_with_token(self, url, data, method, calltimeout):
         auth_key = None
         if self.session:
             auth_key = self.session.headers.get(RestHandler.AUTH_KEY, None)
             if auth_key:
                 self.session.headers[RestHandler.AUTH_KEY] \
                     = cryptor.decode(auth_key)
-        res = self.do_call(url, data, method)
+        res = self.do_call(url, data, method, calltimeout)
```

Comment:
The default value of parameter calltimeout in do_call() is from driver code of HPE 3par, method do_call() is a common method, I think its default value should be also from common module or from  its own.

Reply:
done